codegenerator/cli/src/hbs_templating/contract_import_templates.rs

```diff
@@ -558,4 +555,20 @@ mod test {
 
         assert_eq!(expected_event_keys, actual_event_keys);
     }
+
+    #[test]
+    fn test_get_entity_id_code() {
+        const IS_FUEL: bool = true;
+        assert_eq!(
+            Event::get_entity_id_code(!IS_FUEL, &Language::ReScript),
+            "`${event.chainId->Belt.Int.toString}_${event.block.number->Belt.Int.\
+             toString}_${event.logIndex->Belt.Int.toString}`"
+                .to_string()
+        );
+
+        assert_eq!(
+            Event::get_entity_id_code(IS_FUEL, &Language::TypeScript),
+            "`${event.chainId}_${event.block.height}_${event.logIndex}`".to_string()
+        );
+    }
```

Comment:
It was not adding the Int.toString conversion to each field in the string template.